### DISCUSSION TITLE
MHI2G_ER_AUG22_P3638 MU 0917 FEC 023000EE?

### DISCUSSION THREAD

**Discussion by MASMahmood** [Mr-MIBonk/M.I.B._More-Incredible-Bash#112] [General]
Hi Team,

Sorry I am new to MIB I am from the UK and I have 2017 Audi S5 Sportback and my version is MHI2G_ER_AUG22_P3638.

I am looking to update my maps but my licence key has expired: 02300022

I have been reading on forums and YouTube the key 023000EE get unlimited updates via SD cards.

I have downloaded the M.I.B._More-Incredible-Bash-3.4.1 but I am unable to find a patch folder for MHI2G_ER_AUG22_P3638 is this required to install the FEC key 023000EE?

I have read if you hit the option 'Flash_patched_image |add FECs | CP/AA coding' this may installed FEC key 023000EE - can anyone advise?

I am happy to donate money via PayPal to application team as I would rather give it them than Audi UK who trying to charge me a lot to update my maps.

Thanks for your assistance.

**Comment by harman-f** [comment]
https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/wiki/Prepare-M.I.B-SD-card-&-install-M.I.B#0---start-reading-here-if-you-are-completely-new-to-the-topic

**Reply by MASMahmood** [reply]
Thanks @harman-f for that URL really helped a lot! I will read it and some point have scared me (I know it is at my own risk to this):

Firstly the backup folder (sorry for the stupid question) I assume when you backup your unit in the MIB menu this is where it is stored?

The issue of "Unit not responding - black screen - after reboot" states remove the will need to find the fuse to do this as removing the battery will be a pain as it is located in the boot (also scared this could throw warning lights on my virtual dash).

Also this "No input possible to screen" - once everything is provided how easy is this to fix?

I intend to do this with my car started and my wireless key in the area where the sensor is.

I am going to use a 128GB SD card this one:

https://www.amazon.co.uk/SanDisk-Extreme-SDXC-Memory-Class/dp/B07H9DVLBB/ref=asc_df_B07H9DVLBB/?tag=googshopuk-21&linkCode=df0&hvadid=311059984004&hvpos=&hvnetw=g&hvrand=9635750617181825888&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9046605&hvtargid=pla-608723828365&psc=1

Also once the MIB is uninstalled will I lose my patched FEC licence keys? Will Audi dealership know I have messed with the MMI? I guess I am only messing with the MMI so I they can't really void the warranty on the engine/gearbox/other major components (as I have extended warranty on the car)

@Csall67  Sorry was a mistype from my side MHI2Q_ER_AUG22_P3638 I am having problems downloading the patch collection it is timing out, is there other methods to down this? I know there is "solution" there can use UTorrent for this?

**Reply by olegullikmejdell** [reply]
I also having problem downloading, site is down. I have mhi2_er_aug22_p3241, MU-sofware: 0678, could you please provide me the firmware patch? thanks :)

**Reply by MASMahmood** [reply]
Thanks @Csall67 for the link but I just ended up using the free download manager software that the team advised (this allows you to enter a URL and torrent files for downloading where as my uTorrent only allows torrent files - think for the future would be good to share torrent file).

Will keep updated on how I get on.

**Reply by MASMahmood** [reply]
@harman-f update for you.

Took the leap of faith and installed MIB.

Did have 2 scares on the way (screen seems like it was stuck but progressed to the next stage).

Proceeded with the "Flash patched image | add FECs | CP/AA coding" and this fixed my issue now I am able to install map updates via SD cards (bypasses the licence check).

Please can you confirm the PayPal is correct for donations: https://www.paypal.com/paypalme/chillout1

**Comment by harman-f** [comment]
https://www.paypal.com/paypalme/chillout1 that would be mib2toolbox

Just look up here:
![image](https://user-images.githubusercontent.com/98130152/221672130-1909c1e1-5145-49c2-b004-8ae42c9ff994.png)

**Reply by MASMahmood** [reply]
@harman-f Thanks for confirming.

Gave a donation (I know it's not huge but better than paying Audi UK huge sum of money for something that you guys did for free).
